pulsar-client/src/main/java/org/apache/pulsar/client/impl/TableViewImpl.java

```diff
@@ -104,9 +104,10 @@ private void checkForPartitionsChanges(Timeout timeout) {
         }
 
         start().whenComplete((tw, ex) -> {
-           if (ex != null) {
-               log.warn("Failed to check for changes in number of partitions");
-           }
+            if (ex != null) {
+                log.warn("Failed to check for changes in number of partitions");
```

Comment:
Please report the exception

Reply:
Not sure how to do that - do you mean via Prometheus? Sorry, new to Pulsar.